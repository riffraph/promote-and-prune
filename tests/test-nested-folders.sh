#!/bin/sh

TESTDATAFOLDER=./testNestedFoldersDataset

# test of nested folders

mkdir -p $TESTDATAFOLDER/folder1/folder1a

head -c 8 /dev/urandom > $TESTDATAFOLDER/folder1/folder1a/file1
head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/folder1a/file2


identify.sh $TESTDATAFOLDER
promote.sh $TESTDATAFOLDER


# rm -rf $TESTDATAFOLDER
