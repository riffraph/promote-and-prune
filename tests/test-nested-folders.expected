-- IDENTIFY --
8 ./testNestedFoldersDataset/folder1/folder1a/file2
-- PROMOTE --
TODO
